trusted_os/ctl.go

```diff
@@ -115,6 +119,19 @@ func (ctl *controlInterface) Config(req []byte) (res []byte) {
 	return api.EmptyResponse()
 }
 
+func (ctl *controlInterface) HAB(_ []byte) []byte {
+	srkh, err := hex.DecodeString(ctl.SRKHash)
+	if err != nil {
+		return api.ErrorResponse(fmt.Errorf("built-in SRK hash is invalid: %v", err))
+	}
+	if len(srkh) != sha256.Size {
+		return api.ErrorResponse(errors.New("built-in SRK hash is wrong size"))
+	}
+
+	log.Printf("Would burn SRK hash %x", srkh)
```

Comment:
Make it clearer with a TODO and updated printf that this operation is not yet implemented. If I ran this, confirmed the scary dialog and then got "would burn hash blah" I would feel very confused.